cdcpd/src/cdcpd_node.cpp

```diff
@@ -145,11 +151,11 @@ CDCPD_Moveit_Node::CDCPD_Moveit_Node(std::string const& robot_namespace)
         end_position << node_params.max_rope_length / 2, 0, 1.0;
     }
 
-    // TODO: decide on rope versus cloth configuration here.
-    rope_configuration.initializeTracking(start_position, end_position);
+    initialize_deformable_object_configuration(start_position, end_position);
 
-    std::unique_ptr<CDCPD> cdcpd_new(new CDCPD(nh, ph, rope_configuration.initial.points,
-        rope_configuration.initial.edges,
+    std::unique_ptr<CDCPD> cdcpd_new(new CDCPD(nh, ph,
```

Comment:
I prefer to use `std::make_unique` like this:

```
auto cdcpd_new = std::make_unique<CDCPD>(...)
```

Reply:
You're right, this is much cleaner. I'm able to just use cdcpd = std::make_unique<CDCPD>(...) and avoid the initialization of a temporary variable.